app/views/RequestQuoteView.coffee

```diff
@@ -0,0 +1,90 @@
+RootView = require 'views/core/RootView'
+forms = require 'core/forms'
+TrialRequest = require 'models/TrialRequest'
+AuthModal = require 'views/core/AuthModal'
+
+formSchema = {
+  type: 'object'
+  required: ['name', 'email', 'organization']
+  properties:
+    name: { type: 'string', minLength: 1 }
+    email: { type: 'string', format: 'email' }
+    phoneNumber: { type: 'string' }
+    role: { type: 'string' }
+    organization: { type: 'string' }
+    city: { type: 'string' }
+    state: { type: 'string' }
+    country: { type: 'string' }
+    numberOfStudents: { type: 'string' }
+    educationLevel: {
+      type: 'array'
+      items: { type: 'string' }
+    }
+    notes: { type: 'string' }
+}
+
+module.exports = class RequestQuoteView extends RootView
+  id: 'request-quote-view'
+  template: require 'templates/request-quote-view'
+  
+  events:
+    'submit form': 'onSubmitForm'
+    'click #login-btn': 'onClickLoginButton'
+    'click #signup-btn': 'onClickSignupButton'
+    
+  initialize: ->
+    @trialRequest = new TrialRequest({
+      properties: {
+        email: 'test@gmail.com'
```

Comment:
We shouldn't provide this default email address when logging in via the /teachers/quote page.